pom.xml

```diff
@@ -269,6 +269,7 @@
             <hudson.slaves.NodeProvisioner.initialDelay>0</hudson.slaves.NodeProvisioner.initialDelay>
             <hudson.slaves.NodeProvisioner.MARGIN>50</hudson.slaves.NodeProvisioner.MARGIN>
             <hudson.slaves.NodeProvisioner.MARGIN0>0.85</hudson.slaves.NodeProvisioner.MARGIN0>
+            <jenkins.host.address>${jenkins.host.address}</jenkins.host.address>
```

Comment:
Should this have included `connectorHost`?

Reply:
No need, `connectorHost` is only used for unit tests (https://github.com/jenkinsci/kubernetes-plugin/blob/4cc1f0ad76c9d14d7861f7e4ae712c8f1d7680ca/src/test/java/org/jvnet/hudson/test/JenkinsRuleNonLocalhost.java#L52), not for `hpi:run`